detectors/nuxt.modules.json

```diff
@@ -18,7 +18,7 @@
       "url": "https://http.nuxtjs.org"
     },
     "detectors": {
-      "js": "window.$nuxt?.$options?.context?.app?.$http"
+      "js": "window.$nuxt?.$options?.context?.app?.$http || window.__unctx__?.get('nuxt-app')?.use()?.$http"
```

Comment:
we probably can omit for the Nuxt 2-only modules.

Reply:
Agreed, I tried to check and update which modules have Nuxt 3 support and if the detection would still work by checking for the property.